03 Misc/06 Production Configuration/dev.webpack.config.js

```diff
@@ -1,9 +1,49 @@
+var path = require('path');
+var webpack = require('webpack');
 var webpackMerge = require('webpack-merge');
 var commonConfig = require('./base.webpack.config.js');
 
+var basePath = __dirname;
+
 module.exports = function () {
   return webpackMerge(commonConfig, {
     // For development https://webpack.js.org/configuration/devtool/#for-development
     devtool: 'inline-source-map',
+
+    module: {
+      rules: [
+        {
+          test: /\.scss$/,
+          exclude: /node_modules/,
+          loaders: [
```

Comment:
No, that is the extract text plugin that needs it.

I mean, yes, `use` is better, but *all* examples here use `loader`. So you need to change them all. Will be awkward to have this one good and the other ones wrong.